EU4toV2/Source/Mappers/CountryMappings/CountryMapping.cpp

```diff
@@ -17,6 +17,9 @@ void mappers::CountryMapping::registerKeys()
 	registerKeyword("v2", [this](const std::string& unused, std::istream& theStream) {
 		vic2Tag = commonItems::singleString(theStream).getString();
 	});
+	registerKeyword("hpm", [this](const std::string& unused, std::istream& theStream) {
```

Comment:
this is fine, but at the end of the constructor you should overwrite v2 field with hpm field if configuration:HPM is enabled.

Reply:
Brilliant